src/app/components/User.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+
+class User extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {
+        name: ""
+      }
+    };
+    this.fetchUsers = this.fetchUsers.bind(this);
+  }
```

Comment:
Isn't user more of a model than a React component? We're not "displaying" it for the front end per se.
For later, we probably need this to be a mongoose schema (for MongoDB) for full signup functionality.

Reply:
Yes. we'll need this to be a schema than a react component. I'll remove this change for now